src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -64,3 +53,4 @@ target_link_libraries(crimson-alienstore heap_profiler)
 target_link_libraries(crimson-alienstore ${BLKID_LIBRARIES})
 target_link_libraries(crimson-alienstore ${UDEV_LIBRARIES})
 target_link_libraries(crimson-alienstore crimson)
+target_link_libraries(crimson-alienstore blkdrv)
```

Comment:
@liu-chunmei could you help review this change? i think we deliberately recompile our own version of bluestore. but is it safe to pull the prebuilt blkdrv library from classic osd/rbd?

Reply:
@tchaikov @changchengx in principle, if we didn't modify blkdrv for SEASTAR special, we can link to a prebuild library.  Changcheng, could you test alien store after your changes?  I think we didn't change BlockDevice.h/cc and aio.cc/h KernelDevice.cc/h for SEASTAR. what other files do you put in blkdrv?

Reply:
@changchengx oh, I saw the files you put into blkdrv, I don't think we modify them for SEASTAR and crimson, but please test it to make sure it not influence crimson.

Reply:
@liu-chunmei 
Could you share the test method with me to check it? I'll send an email to you separately.

Reply:
@changchengx send you the test steps by email.

Reply:
@liu-chunmei 
Finish verification, no regression.

Reply:
@tchaikov I have no other concern, please check if your concern is solved.